### ISSUE TITLE
Why cpp

### ISSUE THREAD

**Issue by michoael** [nyadla-sys/whisper.tflite#17]
Any way to use it without cpp
I want use java only

**Comment by vilassn** [comment]
@michoael we are working on adding vad support, realtime transcription and noise processing. Supporting these feature in Java will add significant delay. So using native code to add these feature to avoid processing delay.

Still you can use java only code by initialising whisper engine instance with Java engine instead of Native engine in Whisper.java

**Comment by michoael** [comment]
Okay, thanks.
Now i can use it to transcription in realTime when i speaking withOut end recorder?
 Now i can use it with long wav file?

**Comment by vilassn** [comment]
Yes. We will support realtime transcription and long wav file transcription. It is under implementation.

**Comment by michoael** [comment]
I hope do this very soon, thanks
